elm/wizard.py

```diff
@@ -480,16 +481,17 @@ def __init__(self, db_host, db_port, db_name,
 
         if boto_client is None:
             access_key = os.getenv('AWS_ACCESS_KEY_ID')
-            secret_key = os.getenv('AWS_SECRET_ACCESS_KEY')
-            session_token = os.getenv('AWS_SESSION_TOKEN')
-            assert access_key is not None, "Must set AWS_ACCESS_KEY_ID!"
-            assert secret_key is not None, ("Must set AWS_SECRET_ACCESS_KEY!")
-            assert session_token is not None, "Must set AWS_SESSION_TOKEN!"
-            self._aws_client = boto3.client(service_name='bedrock-runtime',
-                                            region_name='us-west-2',
-                                            aws_access_key_id=access_key,
-                                            aws_secret_access_key=secret_key,
-                                            aws_session_token=session_token)
+            secret = os.getenv('AWS_SECRET_ACCESS_KEY')
```

Comment:
The app uses SSO to access aws resources so the cloud team removed the assert statements since they were breaking the deployment. Doing it this way should preserve functionality locally without breaking the deployment. Let me know what you think.

Reply:
that's fine, you could also raise a warning that would be ignored on the app